src/components/menu.js

```diff
@@ -1,8 +1,19 @@
 import { createElement } from '../lib/dom';
 
-export function createMenu() {
+async function fetchMenu() {
+  const response = await fetch(
+    'https://www.themealdb.com/api/json/v1/1/random.php'
+  );
+  const results = await response.json();
+  const recipe = results.meals[0];
+
+  console.log(recipe);
```

Comment:
Great! The website looks pretty good 👍 .
If you want to use your recipe, you need to return it here.

Reply:
thx :)